### ISSUE TITLE
[DEV] postbootscript "setupntp" failure result in privision failed in sles12.2-ppc64le

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#2762]
After doing ``statelite_installation_flat``, ``statelite_installation_hierarchy_by_ramdisk``, ``statelite_installation_hierarchy_by_nfs`` in sles12.2 ppc64le, the postbootscript ``setupntp`` always failed.

    c910f03c17k29: /usr/sbin/rcntpd ntptimeset
    c910f03c17k29: hwclock: Could not open file with the clock adjustment parameters in it     (/etc/adjtime) for writing: Read-only file system
    c910f03c17k29: hwclock: Drift adjustment parameters not updated.
    c910f03c17k29: grep: /etc/sysconfig/clock: No such file or directory
    c910f03c17k29: Failed to execute operation: Read-only file system
    c910f03c17k29: Mon Mar 27 14:38:12 EDT 2017 postbootscript setupntp return with 1
    c910f03c17k29: Mon Mar 27 14:38:12 EDT 2017 Running postbootscript: otherpkgs
    c910f03c17k29:   Did not install any extra packages.
    c910f03c17k29: Mon Mar 27 14:38:12 EDT 2017 postbootscript otherpkgs return with 0
    c910f03c17k29: node boot failed,reporting status...

That result in the status of provision is "failed"

    [lsdef -l c910f03c17k29 | grep status] Running Time:0 sec
    RETURN rc = 0
    OUTPUT:
        status=failed
        statustime=03-27-2017 14:38:12

**Comment by zet809** [comment]
hi, @cxhong , will you pls help to take a look at this issue?

**Comment by cxhong** [comment]
@hu-weihua , can u show me your osimage? is the default one?  not familiar with statelite, may need @bybai to take look.  it seems permission issue for  file ```/etc/adjtime``, 
````
(/etc/adjtime) for writing: Read-only file system
````

**Comment by bybai** [comment]
Let me reproduce it in sles12.2

**Comment by bybai** [comment]
I can reproduce it in my ENV, let me look into it.

**Comment by bybai** [comment]
Need to add the following in litefile table, to avoid many failures in genimage and fix setupntp failure.
Will commit statelite doc 
```
"ALL","/etc/systemd/system/","tmpfs",,
"ALL","/etc/adjtime","tmpfs",,
```

**Comment by bybai** [comment]
fixed in #2771
